Tools/Pipeline/Common/ContentItem.cs

```diff
@@ -32,6 +32,10 @@ public class ContentItem : IProjectItem
         [Browsable(false)]
         public string OriginalPath { get; set; }
 
+        // For now making this not browsable, until the core functionality works
+        [Browsable(false)]
+        public string Link { get; set; }
```

Comment:
Maybe naming this property `DestinationPath` would be nicer for consistency?

Also remove the comment, it's always gonna be browsable false, think of how linked content is shown in Visual Studio.